packages/access-api/src/service/console.js

```diff
@@ -0,0 +1,35 @@
+import { Console } from '@web3-storage/capabilities'
+import * as Provider from '@ucanto/server'
+import * as API from '../api.js'
+
+/**
```

Comment:
TS only [supports these](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) although jsdoc might have additional ones, but since vscode supports ts ones I don't think others are that useful.

In next version I'll be adding stuff to schema so we'll be defining not just capabilities but invocation responses as well, which should make this into non issue.